Data/Skills/act_int.lua

```diff
@@ -6310,7 +6310,7 @@ skills["OrbOfStorms"] = {
 	statDescriptionScope = "beam_skill_stat_descriptions",
 	castTime = 0.5,
 	preDamageFunc = function(activeSkill, output)
-		activeSkill.skillData.hitTimeOverride = activeSkill.skillData.hitFrequency / (1 + activeSkill.skillModList:Sum("INC", activeSkill.skillCfg, "Speed") / 100)
+		activeSkill.skillData.hitTimeOverride = activeSkill.skillData.hitFrequency / (1 + activeSkill.skillModList:Sum("INC", activeSkill.skillCfg, "Speed") / 100) / activeSkill.skillModList:More(activeSkill.skillCfg, "Speed")
```

Comment:
This works but calcLib.mod() could be used for the entire tail end of this to make it cleaner: https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/dev/Modules/CalcTools.lua#L16-L18

`activeSkill.skillData.hitTimeOverride = activeSkill.skillData.hitFrequency / calcLib.mod(activeSkill.skillModList, activeSkill.skillCfg, "Speed")`